jaws-journey-deploy/orb.yml

```diff
@@ -20,10 +23,48 @@ aliases:
         type: "enum"
         enum: ["sandbox", "nonprod", "prod"]
         default: "sandbox"
-    subproject: &subproject
-      subproject:
+    serviceName: &serviceName
+      serviceName:
         type: "string"
 
+  ignore-master-branch: &ignore-master-branch
+    branches:
+      ignore:
+        - master
+
+  any-cd-pipeline: &any-cd-pipeline
+    filters:
+      branches:
+        only:
+          - master
+          - << pipeline.parameters.sandbox_build_branch >>
+      tags:
+        only: /^\d+\.\d+\.\d+$/
+
+  nonprod-job: &nonprod-job
```

Comment:
I'd rather you put the environment parameters in at this level. 
```yaml
aliases:
  - &all-environments
    filters:
      tags:
        only: /^\d+\.\d+\.\d+$/

  - &sandbox-all
    environment: sandbox
    context: jaws-sandbox
    filters:
      branches:
        ignore:
          - master

  - &sandbox-not-deploy
    environment: sandbox
    context: jaws-sandbox
    filters:
      branches:
        ignore:
          - << pipeline.parameters.sandbox_build_branch >>
          - master

  - &sandbox-deploy
    environment: sandbox
    context: jaws-sandbox
    filters:
      branches:
        only: << pipeline.parameters.sandbox_build_branch >>

  - &nonprod-deploy
    environment: nonprod
    context: jaws-nonprod
    filters:
      branches:
        only: master
```
That way you can't run a nonprod-job with the sandbox or prod environment